lib/Core/Algorithm/src/algorithm.c

```diff
@@ -125,6 +125,8 @@ void InitializeAlgorithmStruct(uint8_t callingFreq)
     gAlgorithm.pointOfInterestB[Y_AXIS] = 0.0;
     gAlgorithm.pointOfInterestB[Z_AXIS] = 0.0;
 
+    gAlgorithm.rtkHeading2magHeading = NAN;
```

Comment:
I think that zero would be a better default in this case. NAN has the option of blowing the filter completely.

Reply:
I was planning to use it as a flag for "do not use rtk heading". That way we could enable/disable it

Reply:
Alrigth.